carmen-salas/short-response.md

```diff
@@ -0,0 +1,79 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer});
+   ```
+* This will log ‘My favorite singer is Omar Apollo’ because the variable is declared with the string ‘Omar Apollo’ and is being concatenated with the string “my favorite singer is”
+
+2. **What happens when we run the following code? Why?**
+   ```javascript
+const favorite = 'Juan Pablo';
+console.log(`Our favorite Marcy Lab family member is ${favorite}!`);
+favorite = 'Maya';
+console.log(`Actually, ${favorite} is my favorite.`);
+   ```
+*It will log `Our favorite Marcy Lab family member is Juan Pablo!` because the varible ‘favorite’ is declared as a const variable which means it can not be changed once it is declared even if it is initiated again.  
```

Comment:
Correct, in essence but let's improve your language.

> It will log `Our favorite Marcy Lab family member is Juan Pablo!` because the ~~varible~~ **variable** ‘favorite’ is declared ~~as a const variable~~ **with the keyword `const`** which means it can not be ~~changed~~ **reassigned** ~~once it is declared even if it is initiated again~~.